# RGB images augmentation configuration
data_aug_conf:
  H: 300
  W: 480
  bot_pct_lim:
    - 0.0
    - 0.0
  final_dim:
    - 256
    - 416

# grid configuration
grid_conf:
  dbound:
  - 0.6
  - 6.4
  - 0.1
  xbound:
  - -6.4
  - 6.4
  - 0.1
  ybound:
  - -6.4
  - 6.4
  - 0.1
  zbound:
  - -3.2
  - 3.2
  - 6.4

# use ImageNet mean and std
img_mean:
- 0.485
- 0.456
- 0.406
img_std:
- 0.229
- 0.224
- 0.225